packages/database/src/migrations/20210816012839-AddProjectPacmossi-modifies-data.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const countryCodes = ['FJ', 'KI', 'NR', 'NU', 'PG', 'SB', 'TV', 'VU'];
```

Comment:
Should 'TO' be included here?

Reply:
@nick-beyondessential thank you, have added!